run_create_s3_srr.py

```diff
@@ -71,22 +77,32 @@ def run_create_s3_srr_bucket(args):
         "Version": "2012-10-17",
         "Statement": [
             {
+                "Effect": "Allow",
                 "Action": [
-                    "s3:Get*",
-                    "s3:ListBucket"
+                    "s3:GetObjectVersionForReplication",
+                    "s3:GetObjectVersionAcl"
                 ],
-                "Effect": "Allow",
                 "Resource": [
-                    "arn:aws:s3:::%s" % origin_bucket_name,
                     "arn:aws:s3:::%s/*" % origin_bucket_name
                 ]
             },
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:ListBucket",
+                    "s3:GetReplicationConfiguration"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::%s" % origin_bucket_name
```

Comment:
@GoldieKMG // 근래에 string format 을 f-string 으로 고치고 있는데 아래와 같이 수정 하시면 좋을 것 같습니다.
```suggestion
                    f"arn:aws:s3:::{origin_bucket_name}"
```